external/fv3fit/fv3fit/reservoir/model.py

```diff
@@ -1,18 +1,60 @@
-from .readout import ReservoirComputingReadout
+import fsspec
```

Comment:
The changes in this module are related to moving the `square_hidden_state_input` option out of the readout and into the model's `.predict` method.